assets/css/_page/_single.scss

```diff
@@ -8,12 +8,18 @@
     line-height: 140%;
   }
 
+.button {
+  margin: 30px auto;
+  display: table;
```

Comment:
I wonder why `table` :)
What about `block` or `flex`? Usually, we shouldn't design elements on tables.

Reply:
Well, I don't know :) I googled it for quite a while and finally it worked. If you say we shouldn't - I won't argue :)

Reply:
Can you please check if a change to block changes anything? If it's the same - use block :)

Reply:
Ok checked - both block and flex doesn't work. Inline-block work though. Is it ok?

Reply:
inline-block is ok :)